bridge/svix-bridge/Cargo.toml

```diff
@@ -28,8 +28,8 @@ tracing-opentelemetry = "0.27.0"
 tracing-subscriber = { workspace = true, features = ["fmt", "json"] }
 # N.b. for newer deno versions (like this) the runtimes must be retained and reused since they will leak memory if you
 # create/drop them.
-deno_core = "0.204.0"
-deno_ast = "0.28.0"
+deno_core = "0.308.0"
```

Comment:
Why this old-ish version? It seems close to the crashy one (though I guess it can't be exactly that wince we have a regression test).

Reply:
This matches the version we use downstream. I don't have a lot of confidence in this library, so for now I thought it would be safer to use a known-ok version from our perspective.

Reply:
Oh, so this _is_ exactly the version that was causing us crashes in bridge before. I'm surprised it would work now.

Reply:
Adding the call to `init_platform` is what fixed it.